Exercise_B/index.html

```diff
@@ -19,16 +19,37 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
+    <header><img src="assets/images/logo.png"></header>
     <!--Banner Jumbotron of the image-->
+    <section class="jumbotron">
+      <div class="container">
+        <div class="row text-center titles">
+          <h1>Beautiful Flowers</h1>
+          <a class="button btn btn-primary" href="#" role="button">Learn More</a>
+        </div>
+      </div>
+    </section>
 
     <!--Four containers with img / text-->
     <!-- 4 containers -->
+    <!-- <div class="container2" > -->
+      <div class="row container2">
+        <div class="col-lg-3 col-md-6 col-sm-12 d-flex align-items-center justify-content-center"><img src="assets/images/flower-icon.png" style="height: 70%; padding-right:50px"><p>Flowers for every <br> kind of situation</p></div>
+        <div class="col-lg-3 col-md-6 col-sm-12 d-flex align-items-center justify-content-center"><img src="assets/images/payment-icon.png" style="height:70%; padding-right:50px"><p>All major payments <br> kinds accepted</br></p></div>
+        <div class="col-lg-3 col-md-6 col-sm-12 d-flex align-items-center justify-content-center"><img src="assets/images/phone-icon.png" style="height:70%; padding-right:50px"><p>Free consultation</p></div>
+        <div class="col-lg-3 col-md-6 col-sm-12 d-flex align-items-center justify-content-center"><img src="assets/images/truck-icon.png" style="height:70%; padding-right:50px"><p>City wide delivery</p>
+      </div>
+    </div>
+    <!-- </div> -->
 
     <!-- Payments Div -->
     <!-- Consultation Div -->
 
     <!-- Delivery Div -->
 
     <!--Scripts required for bootstrap CDN-->
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.3/dist/umd/popper.min.js" integrity="sha384-eMNCOe7tC1doHpGoWe/6oMVemdAVTMs2xqW4mwXrXsW0L84Iytr2wi5v2QjrP/xp" crossorigin="anonymous"></script>
+  
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
   </body>
 </html>

```

Comment:
Amazing Work! This works well and is responsive to different sized width's and the images and text also do resize.